### PR TITLE
Split AssetStatus for separated statuses for BO Asset and DC Asset

### PR DESCRIPTION
**Pull request by mkurek**
- removed AssetStatus choices
- added new choices class for BackOfficeAsset status (BackOfficeAssetStatus)
- added new choices class for DataCenterAsset status (DataCenterAssetStatus)

Connected to #1665

### CODE REVIEW

File: src/ralph/back_office/models.py [hunk 1]
```diff
@@ -12,13 +12,18 @@
...
 from ralph.lib.mixins.models import NamedMixin, TimeStampMixin
 from ralph.lib.transitions.decorators import transition_action
 from ralph.lib.transitions.fields import TransitionField
 from ralph.licences.models import BaseObjectLicence
 
+try:
+    from django.utils.timezone import now as datetime_now
+except ImportError:
+    import datetime
+    datetime_now = datetime.datetime.now
```
Comment (ar4s) [inline src/ralph/back_office/models.py]: Why you catch `ImportError`?

**Comment by szok** [comment]
:ok:

File: src/ralph/reports/views.py [hunk 2]
```diff
@@ -6,16 +7,19 @@
...
-from ralph.assets.models.choices import AssetStatus, ObjectModelType
+from ralph.assets.models.choices import ObjectModelType
 from ralph.back_office.models import BackOfficeAsset
 from ralph.data_center.models.physical import DataCenter, DataCenterAsset
 from ralph.reports.base import ReportContainer
 
+logger = logging.getLogger(__name__)
+
 
 def get_desc(choices_class, key, default='------'):
```
Comment (vi4m) [inline src/ralph/reports/views.py]: Could you be a good scout and add some docstring here?

Comment (vi4m) [inline src/ralph/reports/views.py]: Maybe function signature should change to be less ambiguous

get_desc -> get_descendants?
get_desc -> get_description?

Comment (mkurek) [inline src/ralph/reports/views.py]: done

**Comment by ar4s** [comment]
:ok:

**Pull request merged**